assets/src/edit-story/elements/index.js

```diff
@@ -46,40 +46,40 @@ export const createNewElement = (type, attributes = {}) => {
   };
 };
 
-export const createPage = (attributes = {}) => {
-  const { elements: oldElements, ...rest } = attributes;
+export const createPage = () => {
+  const backgroundElementProps = {
+    // The values of x, y, width, height are irrelevant here, however, need to be set.
+    x: 1,
+    y: 1,
+    width: 1,
+    height: 1,
+    mask: {
+      type: 'rectangle',
+    },
+    isBackground: true,
+    isDefaultBackground: true,
+  };
+  const backgroundElement = createNewElement('shape', backgroundElementProps);
+
+  const newAttributes = {
+    elements: [backgroundElement],
+  };
+
+  return createNewElement('page', newAttributes);
```

Comment:
Should `createNewElement` perhaps complain if a page is being created without a required value? To ensure `createNewElement` is not called directly for a page.

Reply:
I don't think that's necessary. Where would that happen? I don't see someone extending the API and making such a mistake.